products.json

```diff
@@ -380,33 +380,48 @@
                 ]
             },
             {
-                "text": "Portable Gas Detection",
-                "state": {
-                    "opened": false
-                },
-                "children": [
-                    {
-                        "text": "Controller",
-                        "state": {
-                            "opened": false
+               "text": "Portable Gas Detection",
+               "state": {
+                  "opened": false
+               },
+               "children": [
+                  {
+                     "text": "X-am 2800 - 5800",
+                     "state": {
+                        "opened": false
+                     },
+                     "children": [
+                        {
+                           "text": "License Agreements",
+                           "a_attr": { "href": "assets/license_agreements/X-am-2800-5800/License-Agreements.pdf" },
+                           "icon": "fas fa-file-pdf"
+                        }
+                     ]
+                  },
+                  {
+                     "text": "X-dock",
+                     "state": {
+                        "opened": false
+                     },
+                     "children": [
+                        {
+                           "text": "License Agreements",
+                           "a_attr": { "href": "assets/license_agreements/X-dock/License-Agreements_X-dock.pdf" },
```

Comment:
- Document title ("Empfänger (Firma)") needs adaptation
- Bitstream Vera Fonts Copyright is wrong and includes "advertisement" text

Reply:
Attributes cleaned up.